third_party/amd/backend/compiler.py

```diff
@@ -221,7 +267,12 @@ def make_llir(src, metadata, options):
         ## 3. __HIP_FTZ is default to 1 and not exposed as a kernel argument.
         ##    For now it is used as a controller for developers only.
         __HIP_FTZ = True
-        amd.passes.ttgpuir.add_to_llvmir(pm, options.arch, __HIP_FTZ)
+
+        ## enable_buffer_ops determines if using buffer operations while lowering
+        ## memory operations
```

Comment:
Could you also add a comment to mention that the env var is temporary for development purposes until we stabilize better and the goal to remove it after that?